code/xcvm/cosmwasm/contracts/router/Developing.md

```diff
@@ -0,0 +1,104 @@
+# Developing
+
+If you have recently created a contract with this template, you probably could use some
+help on how to build and test the contract, as well as prepare it for production. This
+file attempts to provide a brief overview, assuming you have installed a recent
+version of Rust already (eg. 1.58.1+).
+
+## Prerequisites
+
+Before starting, make sure you have [rustup](https://rustup.rs/) along with a
+recent `rustc` and `cargo` version installed. Currently, we are testing on 1.58.1+.
```

Comment:
not sure if we testing on this version so.

Reply:
Yeah, this document comes from the template itself so better to remove it completely.